dap-python.el

```diff
@@ -216,7 +218,12 @@ overriden in individual `dap-python' launch configurations."
          (plist-put conf :debugServer debug-port)
          (plist-put conf :port debug-port)
          (plist-put conf :hostName host)
-         (plist-put conf :host host)))
+         (plist-put conf :host host)
+         (when (and (not (null justmycode))
+                    (equal justmycode :json-false))
+             (setq debug-options (cl-pushnew "DebugStdLib" debug-options :test #'equal))
+             (plist-put conf :debugOptions debug-options)
```

Comment:
This will create a duplicate if :debugOptions is specified, with this version being overriden (it will show up twice in the JSON communcation). '(:debugOptions ...) -> '(:debugOptions (DebugStdLib . ...) :debugOptions ...)